it/test/utils/WiremockHelper.scala

```diff
@@ -16,12 +16,12 @@
 
 package utils
 
-import akka.actor.ActorSystem
 import com.github.tomakehurst.wiremock.WireMockServer
 import com.github.tomakehurst.wiremock.client.WireMock
 import com.github.tomakehurst.wiremock.client.WireMock._
 import com.github.tomakehurst.wiremock.core.WireMockConfiguration
 import com.github.tomakehurst.wiremock.core.WireMockConfiguration.wireMockConfig
+import org.apache.pekko.actor.ActorSystem
```

Comment:
They named the actorsystem for the implicit val but I don't really see the benefit. Probably can clean it out of the val and import.